eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -156,28 +146,23 @@ stages:
                       - pwsh: |
                           $adjustedName = "${{artifact.name}}" -replace "^azure-", ""
                           Write-Host "##vso[task.setvariable variable=Documentation.Zip]$adjustedName"
-                        displayName: Set Documentation File Name
-                      - template: /eng/pipelines/templates/steps/stage-artifacts.yml
-                        parameters:
-                          SourceFolder: ${{parameters.ArtifactName}}
-                          TargetFolder: ${{artifact.safeName}}
-                          PackageName: ${{artifact.name}}-[0-9]*.[0-9]*.[0-9]*.tgz
+                        displayName: Set Documentation File Name                        
                       - template: /eng/pipelines/templates/steps/stage-artifacts.yml
```

Comment:
This is where stage-artifact is still used to copy over all document artifact into upload directory. I will make a change to use copy script instead of stage-artifact template so we can get rid of it.